### PR TITLE
add aliyun_ftasr api

### PR DESCRIPTION
**Pull request by Lizerui9926**

### CODE REVIEW

File: models/aliyun_ftasr_api_zh/asr_sdk.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    if statusText == STATUS_SUCCESS :
+        print ("录音文件识别成功！")
+        asr_result = getResponse[KEY_RESULT]
+        return asr_result["Sentences"][0]["Text"]
+    else :
+        print ("录音文件识别失败！")
+        return ""
+
+if __name__ == "__main__":
+    accessKeyId = "您的AccessKey Id"
```
Comment (dophist) [inline models/aliyun_ftasr_api_zh/asr_sdk.py:110]: Credentials should not be filled in code like these.  Please refer to aliyun_api_zh model, load these credentials from secret files.

File: models/aliyun_ftasr_api_zh/docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:20.04
+LABEL maintainer="jerry.jiayu.du@gmail.com"
```
Comment (dophist) [inline models/aliyun_ftasr_api_zh/docker/Dockerfile:2]: fix maintainer email address

File: models/aliyun_ftasr_api_zh/read_oss_file.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+if __name__ == "__main__":
+    if len(sys.argv) != 3:
+        sys.stderr.write("read_oss_file.py <wav_scp> <oss_out_scp>\n")
+        exit(-1)
+
+    idxList = []
+    wavFile = codecs.open(sys.argv[1], 'r',  'utf8') 
+    # wav OSS存放地址
+    ossPath = "oss://example/wav/"
```
Comment (dophist) [inline models/aliyun_ftasr_api_zh/read_oss_file.py:22]: I'm kinda confused with this `ossPath`:
1. who own this bucket? is this bucket accessible from public? Some of SpeechIO test sets are not open-sourced yet, uploading files to a public oss path will leak those test sets.
2. even if this is an oss path filled by leaderboard caller, does it mean that leaderboard caller need to create this bucket and directory manually from Aliyun web-console? Things like these steps can not be automated within an automate pipeline.

Comment (dophist) [reply models/aliyun_ftasr_api_zh/read_oss_file.py:22]: Also, how is this OSS bucket charged? Does this mean the leaderboard caller need to maintain this bucket and pay for uploading plus 7x24 storage fees?

File: models/aliyun_ftasr_api_zh/asr_sdk.py [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+import time
+import sys
+import codecs
+from aliyunsdkcore.acs_exception.exceptions import ClientException
+from aliyunsdkcore.acs_exception.exceptions import ServerException
+from aliyunsdkcore.client import AcsClient
+from aliyunsdkcore.request import CommonRequest
+
+accessKeyId = ""
+with open('accessKeyId', 'r') as f:
```
Comment (dophist) [inline models/aliyun_ftasr_api_zh/asr_sdk.py:18]: please rename the credential secret to be consistent with https://github.com/SpeechColab/Leaderboard/blob/master/models/aliyun_api_zh/update_token.py#L16

File: models/aliyun_ftasr_api_zh/asr_sdk.py [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+    accessKeyId = f.readline().strip()
+
+accessKeySecret = ""
+with open('accessKeySecret', 'r') as f:
```
Comment (dophist) [inline models/aliyun_ftasr_api_zh/asr_sdk.py:22]: same kind of renaming as above, refer to https://github.com/SpeechColab/Leaderboard/blob/master/models/aliyun_api_zh/update_token.py#L19

**Review by dophist — CHANGES_REQUESTED** [review]
Two things left: 

* For security reason, please remove `ossutil64` from your model image, and download it *from the official Aliyun url* in the code.  This avoid the docker env running a totally untrusted binary.  You can assume the testing machine is connected to domestic internet, and use things like wget, but make sure you add `wget` dependency in your Dockerfile.

* Also make sure you can successfully validate your model submission on your own machine with your own Aliyun credential, as described in HOW_TO_SUBMIT.md

**Review by dophist — COMMENTED** [review]
Looks good to me now.  Did you validate the submission on your own machine with you own Aliyun credential ?

**Comment by Lizerui9926** [comment]
> Looks good to me now. Did you validate the submission on your own machine with you own Aliyun credential ?

Yes, we have verified the pipeline on MINI and UNLOCKED SpeechIO test sets.

**Pull request merged**